packages/mask/src/components/shared/ApplicationBoard.tsx

```diff
@@ -301,53 +273,32 @@ const ApplicationEntryStatusContext = createContext<ApplicationEntryStatusContex
     shouldVerifyNextId: undefined,
     currentPersonaPublicKey: undefined,
     currentSNSConnectedPersonaPublicKey: undefined,
-    personaConnectAction: undefined,
-    personaNextIDReset: undefined,
+    personaAction: undefined,
     isLoading: false,
 })
 
 function ApplicationEntryStatusProvider(props: PropsWithChildren<{}>) {
-    const personaConnectStatus = useCurrentPersonaConnectStatus()
-    const nextIDConnectStatus = useNextIDConnectStatus(true)
-
-    const {
-        value: ApplicationCurrentStatus,
-        retry,
-        loading: personaAgainstSNSConnectStatusLoading,
-    } = usePersonaAgainstSNSConnectStatus()
-
-    useEffect(() => {
-        retry()
-        nextIDConnectStatus.reset()
-        return MaskMessages.events.currentPersonaIdentifier.on(() => {
-            retry()
-            nextIDConnectStatus.reset()
-        })
-    }, [])
-
-    const personaNextIDReset = useCallback(() => {
-        nextIDConnectStatus.reset()
-        verifyPersona(personaConnectStatus.currentPersona?.identifier)()
-    }, [nextIDConnectStatus, personaConnectStatus])
+    const { value: personaConnectStatus, loading: personaStatusLoading } = useCurrentPersonaConnectStatus()
+    const { value: ApplicationCurrentStatus, loading: personaAgainstSNSConnectStatusLoading } =
+        usePersonaAgainstSNSConnectStatus()
 
     const { isSNSConnectToCurrentPersona, currentPersonaPublicKey, currentSNSConnectedPersonaPublicKey } =
         ApplicationCurrentStatus ?? {}
 
     return (
         <ApplicationEntryStatusContext.Provider
             value={{
-                personaConnectAction: personaConnectStatus.action ?? undefined,
-                personaNextIDReset,
+                personaAction: personaConnectStatus.action,
                 isPersonaCreated: personaConnectStatus.hasPersona,
                 isPersonaConnected: personaConnectStatus.connected,
-                isNextIDVerify: nextIDConnectStatus.isVerified,
+                isNextIDVerify: personaConnectStatus.verified,
```

Comment:
What's the difference with `isPersonaConnected`?

Reply:
`Verified` mean  bound on nextID.
`Connected` mean on local.